tests/test_SdssShape.py

```diff
@@ -129,6 +139,47 @@ def testMeasureBadPsf(self):
             self._checkShape(result, record)
             self.assertTrue(result.getFlag(lsst.meas.base.SdssShapeAlgorithm.PSF_SHAPE_BAD.number))
 
+    def testMonteCarlo(self):
+        """Test an ideal simulation, with no noise.
+
+        Demonstrate that:
+
+        - We get exactly the right answer, and
+        - The reported uncertainty agrees with a Monte Carlo test of the noise.
+        """
+        algorithm, schema = self.makeAlgorithm()
+        # Results are RNG dependent; we choose a seed that is known to pass.
+        exposure, cat = self.dataset.realize(0.0, schema, randomSeed=3)
+        record = cat[1]
+        instFlux = record["truth_instFlux"]
+        algorithm.measure(record, exposure)
+        for suffix in ["xx", "yy", "xy"]:
+            self.assertFloatsAlmostEqual(record.get("truth_"+suffix),
+                                         record.get("base_SdssShape_"+suffix), rtol=1E-4)
+
+        for noise in (0.0001, 0.001,):
+            nSamples = 1000
+            catalog = lsst.afw.table.SourceCatalog(cat.schema)
+            for i in range(nSamples):
+                # By using ``i`` to seed the RNG, we get results which
+                # fall within the tolerances defined below. If we allow this
+                # test to be truly random, passing becomes RNG-dependent.
+                exposure, cat = self.dataset.realize(noise*instFlux, schema, randomSeed=i)
+                record = cat[1]
+                algorithm.measure(record, exposure)
+                catalog.append(record)
+
+            catalog = catalog.copy(deep=True)
+            for suffix in ["xx", "yy", "xy"]:
+                shapeMean = np.mean(catalog["base_SdssShape_"+suffix])
+                shapeErrMean = np.nanmean(catalog["base_SdssShape_"+suffix+"Err"])
+                shapeStandardDeviation = 0.5*(np.nanpercentile(catalog["base_SdssShape_"+suffix], 84)
```

Comment:
Maybe call this `shapeInterval68`?  I think it's only equivalent to the standard deviation for a Gaussian distribution.